betelgeuse.py

```diff
@@ -41,6 +42,7 @@
 # Avoid SSL errors
 ssl._create_default_https_context = ssl._create_unverified_context
 
+INVALID_TEST_PLAN_CHARS_REGEX = re.compile(r'[\\/.:"<>|~!@#$?%^&\'*()+`,= ]')
```

Comment:
@elyezer actually, it is not the same. It has an additional space in it :) But let me see if I can do something here.